app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -105,10 +105,11 @@ class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, clien
       search.commodityCode.map(qb.unbind("commodity_code", _)),
       search.decisionDetails.map(qb.unbind("decision_details", _)),
       search.status.map(_.map(s => qb.unbind("status", s.toString)).mkString("&")),
+      search.applicationType.map(_.map(s => qb.unbind("application_type", s.toString)).mkString("&")),
       search.keywords.map(_.map(k => qb.unbind("keyword", k)).mkString("&"))
     ).filter(_.isDefined).map(_.get)
 
-    val url = s"${appConfig.bindingTariffClassificationUrl}/cases?application_type=BTI&${(reqParams ++ optParams).mkString("&")}&page=${pagination.page}&page_size=${pagination.pageSize}"
+    val url = s"${appConfig.bindingTariffClassificationUrl}/cases?${(reqParams ++ optParams).mkString("&")}&page=${pagination.page}&page_size=${pagination.pageSize}"
```

Comment:
Wonder if it'd be worth pulling in the changes from branch for DIT-1078 and expanding/using the buildQuery method there?

Reply:
Is DIT-1078 ready for merging?

Reply:
It is unless anyone else has any comments on it

Reply:
DIT-1078 is merged in now

Reply:
I'm not sure the builder adds much for this instance (I can understand why it helps with the queues though)